src/drive/payments/index.ts

```diff
@@ -84,6 +85,14 @@ export class Payments {
     return this.client.get<DisplayPrice[]>('/prices', this.headers());
   }
 
+  public getCoupon(): Promise<CouponAvailable> {
+    return this.client.get('/request-coupon', this.headers());
+  }
+
+  public applyCoupon(coupon: string): Promise<CouponAvailable> {
```

Comment:
Do not expect to receive a coupon from `getCoupon()` and to have to send it on `applyCoupon()`

We are handling this internally. Ref: https://github.com/internxt/payments/blob/9c271d0843a511fbbdcaa800977a1290abeb5ccc/src/controller.ts#L162

Reply:
Already removed